dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Loads/LoadShedding.mo

```diff
@@ -0,0 +1,56 @@
+within Dynawo.Electrical.Controls.Loads;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model LoadShedding "Load shedding model using a CombiTimeTable"
+  import Modelica;
+
+  parameter String LoadSheddingProfileFileName = "NoName" "Name of the file where the table describing the load shedding profile over time is stored";
+  parameter String LoadSheddingProfileTableName = "NoName" "Name of the table describing the load shedding profile over time";
+
+  Modelica.Blocks.Interfaces.RealInput deltaPNom(start = 0) "Nominal deltaP required by the load shedding" annotation(
```

Comment:
Instead of deltaP, deltaQ, shouldn't you write "active load variation", "reactive load variation" in the description?

Reply:
done